graph/run-dynamic.go

```diff
@@ -106,6 +106,9 @@ func (gt *GraphThread[V, E, M, N]) checkCommandsDynamic(blockTop, bspSync, block
 	case BLOCK_TOP:
 		*blockTop = true
 		gt.Response <- ACK
+	case BLOCK_TOP_ASYNC:
+		*blockTop = true
+		// No ack needed.
```

Comment:
The global relabeling requires blocking the topological events (otherwise the super step would not terminate). `BLOCK_TOP` doesn't work because it needs ACKs which are also used to indicate the completion of an epoch.

Reply:
Makes sense to me!